fsthttp/request.go

```diff
@@ -230,6 +254,14 @@ func (req *Request) Clone() *Request {
 	}
 }
 
+// CloneWithBody returns a copy of the request, with the Body field set
+// to the provided io.Reader.  Its URL will have a nil User field.
+func (req *Request) CloneWithBody(body io.Reader) *Request {
```

Comment:
I agree this could be removed now, perhaps by including the two lines of code into the doc of `SetBody`.